### PR TITLE
Ran basic perltidy on all the perl scripts

### PR DESCRIPTION
**Pull request by agraubert**
I just ran perltidy on the pl files, and let it do its thing.
 #16

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,3 +18,4 @@ nytprof.out
 *.o
 *.bs
 /_eumm/
+.DS_STORE
```
Comment (susannasiebert) [inline .gitignore]: I don't think we want to commit this change to your .gitignore

**Comment by susannasiebert** [comment]
+1 aside from the change to `.gitignore`

File: .gitignore (hunk 1 above)
Comment (agraubert) [inline .gitignore]: I'll make a commit to revert it and add that into this pr then

**Comment by jasonwalker80** [comment]
+1

**Pull request merged**